.env

```diff
@@ -10,3 +10,11 @@ HYPERDX_APP_URL=http://localhost
 HYPERDX_LOG_LEVEL=debug
 OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318 # port is fixed
 
+# AWS configuration
+ECS_PROFILE_NAME=hyperdx
+ECS_SERVICE_NAME=hyperdx
+ECS_CLUSTER_NAME=hyperdx
+AWS_REGION=us-east-1
+ECS_LAUNCH_TYPE=FARGATE
+HYPERDX_APP_ALB_PORT=443
+HYPERDX_APP_ALB_URL=https://localhost
```

Comment:
since this is in README already, I think we can either remove them or add them to `/aws/.env.aws`